test/feature/reverse_routing.feature

```diff
@@ -26,7 +27,7 @@ Feature: reverse routing
       GET '/users/:userId/posts/:id', to: 'posts#show', as: 'userPost'
       """
     And an exprestive app using defaults
-    Then I have a routing helper "res.locals.paths.userPost({userId: 1, id: 2})" that returns "/users/1/posts/2"
+    Then I have a routing helper "res.locals.routes.userPost({userId: 1, id: 2})" that returns "/users/1/posts/2"
```

Comment:
Since you support camelCase and snake_case above, maybe also verify that `res.locals.routes.userPost({user_id: 1, id: 2})` works here.